daemon/daemon.go

```diff
@@ -607,7 +595,7 @@ func NewDaemon(c *Config) (*Daemon, error) {
 		consumableCache:   policy.NewConsumableCache(),
 		policy:            policy.NewPolicyRepository(),
 		ignoredContainers: make(map[string]int),
-		buildEndpointChan: make(chan *endpoint.Request, common.EndpointsPerHost),
+		buildEndpointChan: make(chan *endpoint.Request, 512),
```

Comment:
Discussed a bit on slack. I've put the original channel size in place again. We will look at this issue in a separate PR.